backend/danswer/configs/constants.py

```diff
@@ -1,5 +1,6 @@
 from enum import Enum
 
+BACKEND_VERSION = "1.0.0"
```

Comment:
A standard is to have it in the __init__.py by the entrypoint which is main.py.

See here:
<img width="710" alt="image" src="https://github.com/danswer-ai/danswer/assets/32520769/de699c0d-1b8f-4831-b753-7ff22d1dc89c">

Be sure to have it be able to be read from env and prioritize that value so that it can be used with the build process